### DISCUSSION TITLE
RETWEET PROBLEM

### DISCUSSION THREAD

**Discussion by girizgah** [IBAX-io/go-ibax#1687] [Bug Submission]
Even though I tried the last task 50 times, it doesn't approve. If the authorities contact me, I can also post the video. I hope this minor glitch that will affect the ranking will be fixed soon.

**Comment by tanghua2008** [comment]
very good

**Comment by muco333** [comment]
https://twitter.com/KavusMucahit/status/1488035141880619009

give support for this issue.

**Comment by hizen1984** [comment]
yogunluk diyorlarda , sabah sabah ne yogunlugu ?
